src/helm/benchmark/metrics/basic_metrics.py

```diff
@@ -278,6 +278,22 @@ def tokenize(self, text: str) -> List[str]:
     return partial(rouge_score, scorer=scorer, rouge_type=rouge_type)
 
 
+def chinese_math_exact_match_indicator(gold: str, pred: str) -> float:
```

Comment:
I recommend a different name: we are not using indicator here.

Reply:
I changed the name to `chinese_math_result_match`